CorsixTH/Lua/entities/patient.lua

```diff
@@ -1028,9 +1039,12 @@ function Patient:setTile(x, y)
           end
         end
       end
-      self.litter_countdown = nil
+
+      -- Always give new litter to drop.
+      self.litter_countdown = math.random(30, 150)
```

Comment:
The idea was that people staying longer in the hospital and dropping litter have less litter to drop later, but otherwise, not really.

Splitting between the first and next literings make it feasible to hit bigger / less efficient hospitals harder of softer. Not sure what the right values are here, likely they need a little tweaking later.